### PR TITLE
feat(music-player-album): default album art image when no music is playing

### PR DESCRIPTION
**Pull request by Krr0ptioN**
## Description

This PR aims to resolve the issue of no album art when no music is selected. 

### Screenshot of the problem

image

### Solution

image

### CODE REVIEW

File: ui/src/assets/img/default-music.jpg
Comment (code-by-sia) [inline ui/src/assets/img/default-music.jpg:1]: Could we use an SVG files instead?

**Comment by Krr0ptioN** [comment]
What do you think about red bg for the default picture?

File: ui/src/assets/img/default-music.jpg
Comment (Krr0ptioN) [reply ui/src/assets/img/default-music.jpg:1]: SVG files do not load, 

image

I check the parcel docs. I add the @parcel/transformer-svg-react deps, however, there was effect despite the configuration.

Parcel config:

```json
{
  "extends": "@parcel/config-default",
  "transformers": {
        "jsx:*.svg": ["...", "@parcel/transformer-svg-react"],
        "jsx:*": ["..."],
        "*.svg": ["...", "@parcel/transformer-svg-react"]
  }
}
```

Comment (code-by-sia) [reply ui/src/assets/img/default-music.jpg:1]: We don't nede to use a static file, instead an jsx SvgIcon would be enough.

**Review by code-by-sia — APPROVED** [review]

**Pull request merged**